File: Ola/Array/Day 3/removeElement.js
//Solution
let nums = [7, 9, 9, 7, 2]
let val = 7
function removeElement(nums, val) {

  for (let i=0; i<nums.length; i++){
  if(nums[i]===val){
      nums.splice(i,1);
      i--
    }
  }
  return nums.length;
};
console.log(removeElement(nums, val));

File: Tope/Day3/removeElement.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        if val == []:
            return 0
        else:
            i = 0
            j = 0
            while j < len(nums):
                if nums[j] == val:
                    j += 1
                else:
                    nums[i] = nums[j]
                    i += 1
                    j += 1

            return len(nums[0:i])

input_list = [3,12,14,17]
ob1 = Solution()
print(ob1.removeElement(input_list, 15))